src/lib/hooks/gun/useGunProfile.js

```diff
@@ -1,6 +1,6 @@
 import { useEffect, useRef, useState } from 'react'
 import { get, result } from 'lodash'
-import userStorage from '../../gundb/UserStorage'
+import userStorage from '../../userStorage/UserStorage'
```

Comment:
1. restore gundb
2. need to create a new hook not dependant on gun and replace usages of useGunProfile